content/components/tooltip.mdx

```diff
@@ -3,42 +3,239 @@ title: Tooltip
 description: Tooltips add additional context to other UI elements and appear on mouse hover or keyboard focus.
 reactId: tooltip
 railsIds:
-- Primer::Alpha::Tooltip
-- Primer::Tooltip
+  - Primer::Alpha::Tooltip
+  - Primer::Tooltip
 cssId: tooltip
 ---
 
 import ComponentLayout from '~/src/layouts/component-layout'
-import {Link, Text} from '@primer/react'
+import {Link, Text, Box} from '@primer/react'
 import {Link as GatsbyLink} from 'gatsby'
 export default ComponentLayout
 
+<Note variant="danger">
+  <Text sx={{display: 'block', fontWeight: 'bold', mb: 2}}>⚠️ Usage Warning ⚠️</Text>
+  <Text>
+    Tooltips as a UI pattern should be the last resort for conveying information as they are hidden by default and often
+    with zero or little visual indicator of its existence. See tooltip alternaitves below before using this component.
+  </Text>
+</Note>
 
 ## Usage
 
-The tooltip component adds a tooltip to add context to interactive elements on the page.
-
 <img
   width="960"
   role="presentation"
-  src="https://github.com/primer/design/assets/586552/787873b7-d496-454c-b669-0c2400ef503f"
+  src="https://github.com/primer/primitives/assets/18661030/172aa8f6-2273-4719-bb63-8dd678ffcbc9"
   alt="three examples of tooltips on interactive elements"
 />
 
-## Accessibility
+- Reserve tooltips for components such as icon buttons.
+- Keep the tooltip text minimal.
+- **Never** include tooltips on non-interactive components (`div`, `span`, `p`).
```

Comment:
```suggestion
- **Never** include tooltips on non-interactive components (`div`, `span`, `p`) because it will not be accessible for keyboard or screen reader users.
```